bin/guard

```diff
@@ -2,31 +2,114 @@
 
 require "pathname"
 
-default = Pathname(ENV.fetch("BUNDLE_GEMFILE", "Gemfile"))
-relative_to_binstub = Pathname(__FILE__).realpath + "../../Gemfile"
-gemfile = [default, relative_to_binstub].detect(&:exist?)
-
-if gemfile && !ENV["RUBYGEMS_GEMDEPS"]
-  ENV["BUNDLE_GEMFILE"] = gemfile.to_s
-  require "rubygems"
-  require "bundler/setup"
-else
-  require "rubygems"
-end
+class GuardReloader
+  class Config
+    def using_rubygems?
+      ENV["RUBYGEMS_GEMDEPS"]
+    end
+
+    def setup_rubygems_for_deps
+      require "rubygems"
+    end
+
+    def current_bundler_gemfile
+      ENV["BUNDLE_GEMFILE"]
+    end
+
+    def using_bundler?
```

Comment:
Presence of that variable suggests bundler is wanted/used, because that variable is used by Bundler. If that variable is not needed, it should not be set or be explicitly removed.